packages/components/inputs/localized-rich-text-input/src/editor.tsx

```diff
@@ -348,6 +350,11 @@ const Editor = forwardRef((props: TEditorProps, forwardedRef) => {
                 </RightColumn>
               )}
             </Row>
+            {props.additionalInfo && (
+              <LeftColumn>
+                <InfoMessage>{props.additionalInfo}</InfoMessage>
+              </LeftColumn>
+            )}
```

Comment:
I think rendering the additional info should be placed where the errors and warning are (around line 308) so no new Row component is needed.

Reply:
It was like that before but it would not align with this design
<img width="715" alt="Screenshot 2024-02-01 at 12 35 13" src="https://github.com/commercetools/ui-kit/assets/42547802/52ef5a1d-7e0d-497f-935d-94190ebf9633">

Reply:
Oh! I missed that one 🤦 

Thanks for the explanation 👍